rero_ils/modules/documents/jsonschemas/documents/document_genre_form-v0.0.1.json

```diff
@@ -6,83 +6,27 @@
     "minItems": 1,
     "items": {
       "type": "object",
-      "title": "Genre, form",
-      "oneOf": [
-        {
-          "title": "Link to concept entity",
-          "type": "object",
-          "additionalProperties": false,
-          "propertiesOrder": [
-            "$ref"
-          ],
-          "required": [
-            "$ref"
-          ],
-          "properties": {
-            "$ref": {
-              "title": "Concept",
-              "type": "string",
-              "pattern": "^https://mef.rero.ch/api/concepts/rero/.*?$",
-              "form": {
-                "remoteTypeahead": {
-                  "type": "mef-concepts",
-                  "enableGroupField": true
-                },
-                "templateOptions": {
-                  "itemCssClass": "col-lg-12"
-                }
-              }
-            }
-          }
-        },
-        {
-          "title": "Concept (local)",
-          "type": "object",
-          "additionalProperties": false,
-          "propertiesOrder": [
-            "type",
-            "term",
-            "identifiedBy",
-            "source"
-          ],
-          "required": [
-            "type",
-            "term"
-          ],
-          "properties": {
-            "source": {
-              "title": "Source",
-              "description": "Source of the subject, for instance rero.",
-              "type": "string"
-            },
-            "type": {
-              "title": "Type",
-              "type": "string",
-              "default": "bf:Topic",
-              "enum": [
-                "bf:Topic"
-              ],
-              "form": {
-                "type": "selectWithSort",
-                "options": [
-                  {
-                    "label": "bf:Topic",
-                    "value": "bf:Topic"
-                  }
-                ]
-              }
+      "title": "Entities",
+      "description": "Topic (including genre/form), place, temporal, person, family or corporate body (including conferences). Always create a link to IdRef or GND, if possible.",
```

Comment:
"Genre or form of the document. Always create a link to IdRef or GND, if possible."